Sources/RDeviceIdentifier/RDeviceIdentifier.swift

```diff
@@ -0,0 +1,179 @@
+import UIKit.UIDevice
+import CommonCrypto
+#if !canImport(RSDKUtils)
+import RLogger // Required in SPM version
+#endif
+
+@objc public final class RDeviceIdentifier: NSObject {
+    private static let keychain = RDeviceIdentifierKeychain()
```

Comment:
Should a keychain protocol be injected into a RDeviceIdentifier initializer?

Reply:
We'd have to `public`ly expose a Keychain protocol making this API a bit more tedious to use. Maybe the only way if we need to inject the accessGroup/service params somewhere.

Reply:
For now, that's fine to keep everything static then. We'll see how we can improve this class in an other ticket.

Reply:
For now, this is fine.